js/_actions.js

```diff
@@ -63,6 +63,22 @@ export function failRevisionsGet(err) {
 	};
 }
 
+export const SUCCEED_DEPLOY_HISTORY_GET = 'SUCCEED_DEPLOY_HISTORY_GET';
```

Comment:
You might want to get a STARTED_* action, so you can display a loader indicator. For example during initial load and for pagination.